msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/perf/TestCasePerf.java

```diff
@@ -56,89 +56,90 @@
-                                .loginHint(mLoginHint)
-                                .sessionExpected(false)
-                                .consentPageExpected(false)
-                                .speedBumpExpected(false)
-                                .build();
-
-                        new AadPromptHandler(promptHandlerParameters)
-                                .handlePrompt(username, password);
-                    }
-                }
-        );
-
-        interactiveRequest.execute();
-        latch.await(TokenRequestTimeout.SHORT);
-
-        final IAccount account = getAccount();
-        codeMarkerManager.setEnableCodeMarker(true);
-        //Setting up scenario code. 100 -> Non Brokered, 200 -> Brokered
-        codeMarkerManager.setPrefixScenarioCode(PerfConstants.ScenarioConstants.SCENARIO_NON_BROKERED_ACQUIRE_TOKEN_SILENTLY);
-
-        for(int i = 0; i < numberOfOccurrenceOfTest; i++) {
-            codeMarkerManager.clearMarkers();
-            final TokenRequestLatch silentLatch = new TokenRequestLatch(1);
-
-            final AcquireTokenSilentParameters silentParameters = new AcquireTokenSilentParameters.Builder()
-                    .forAccount(account)
-                    .fromAuthority(account.getAuthority())
-                    .forceRefresh(true)
-                    .withScopes(Arrays.asList(mScopes))
-                    .withCallback(successfulSilentCallback(silentLatch))
-                    .build();
-
-            mApplication.acquireTokenSilentAsync(silentParameters);
-            silentLatch.await(TokenRequestTimeout.SILENT);
-            try {
-                FileAppender fileAppender = new FileAppender(outputFilenamePrefix + i + ".txt", new SimpleTextFormatter());
-                fileAppender.append(codeMarkerManager.getFileContent());
-                CommonUtils.copyFileToFolderInSdCard(
-                        fileAppender.getLogFile(),
-                        "automation"
-                );
-            } catch (IOException e) {
-                throw new AssertionError("IOException while writing Perf data file");
-            }
-
-            // If this is not the last iteration, then we need either to clear cache of access token manually or wait for 30 seconds.
-            if(i < numberOfOccurrenceOfTest - 1) {
-                // CommandDispatcherHelper.clear();
-                try {
-                    // Sleep for 30 seconds so that the cache access token cache is cleared.
-                    Thread.sleep(TimeUnit.SECONDS.toMillis(30));
-                } catch (InterruptedException e) {
-                    throw new AssertionError("Interrupted while sleeping for 30 seconds so that old access token could have been out of chache");
-                }
-            }
-        }
-        codeMarkerManager.clearMarkers();
-        codeMarkerManager.setEnableCodeMarker(false);
+//        CodeMarkerManager codeMarkerManager = CodeMarkerManager.getInstance();
```

Comment:
Why are we checking in this commented code?

Reply:
It was causing some build errors locally but I've reverted it for this PR